src/test/java/com/puppycrawl/tools/checkstyle/bdd/InlineConfigParser.java

```diff
@@ -517,6 +534,14 @@ private static void setProperties(String inputFilePath, Configuration module,
         }
     }
 
+    private static void setViolationsForGoogleConfig(
+            TestInputConfiguration.Builder inputConfigBuilder,
+            List<String> lines) throws CheckstyleException {
```

Comment:
please inline this method to where it is used, it is too small

Reply:
done

https://github.com/checkstyle/checkstyle/blob/a0f82691156c959fbb7dccd991c3e5c4f83022e1/src/test/java/com/puppycrawl/tools/checkstyle/bdd/InlineConfigParser.java#L227-L233